### PR TITLE
Update release and workflow documentation

### PR DESCRIPTION
**Pull request by MarekMichali**
**Description**

Changes proposed in this pull request:

- Update release docs
- Update workflow docs

**Related issue(s)**

See #831

### CODE REVIEW

File: docs/contributor/04-10-workflows.md [hunk 1]
```diff
@@ -84,3 +88,24 @@ The workflow:
...
+
+| Parameter name  | Required | Description                                                          |
+| ------------- | ------------- |----------------------------------------------------------------------|
+| **last-k3s-versions**  | no  | number of most recent k3s versions to be used for tests, default = `1` |
+| **release**  | no  | determines if the workflow is called from release, default = `true` |
+| **version**  | no  | chart version, default = `0.0.0.0` |
+
+
+The workflow:
+- Checks if the KEB chart is rendered by Helm
```
Comment (jaroslaw-pieszka) [inline docs/contributor/04-10-workflows.md:105]: `can be rendered`? `is rendered successfully`?

Comment (MarekMichali) [reply docs/contributor/04-10-workflows.md]: Is rendered successfully. Changed

File: docs/contributor/04-10-workflows.md [hunk 2]
```diff
@@ -84,3 +88,24 @@ The workflow:
...
+- Fetches the **last-k3s-versions** tag versions of k3s releases 
+- Prepares the **last-k3s-versions** k3s clusters with the Docker registries using the list of versions from the previous step
+- Creates required namespaces
+- Installs required dependencies by the KEB charts
+- Installs the KEB chart in the k3s cluster using helm install
```
[the remark this answers is not in the document]
Comment (MarekMichali) [reply docs/contributor/04-10-workflows.md]: You are right.

File: docs/contributor/04-10-workflows.md [hunk 3]
```diff
@@ -84,3 +88,24 @@ The workflow:
...
+| **release**  | no  | determines if the workflow is called from release, default = `true` |
+| **version**  | no  | chart version, default = `0.0.0.0` |
+
+
+The workflow:
+- Checks if the KEB chart is rendered successfully by Helm
+- Fetches the **last-k3s-versions** tag versions of k3s releases 
+- Prepares the **last-k3s-versions** k3s clusters with the Docker registries using the list of versions from the previous step
+- Creates required namespaces
+- Installs required dependencies by the KEB
```
Comment (IwonaLanger) [inline docs/contributor/04-10-workflows.md:109]:
```suggestion
- Installs required dependencies by the KEB chart
```

Comment (IwonaLanger) [reply docs/contributor/04-10-workflows.md:109]: If the word chart should not be used, please remove also `the` (from `the KEB`).

**Review by IwonaLanger — APPROVED** [review]

**Review by jaroslaw-pieszka — APPROVED** [review]

**Pull request merged**